public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -56,7 +56,7 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
       // Sets 'compare from' version from the state if available
       // Sets the first revision from the list otherwise
       vm.fromSelect.selected = cascade(
-        _.bind(dataUtil.findVersionItem, null, items, state.from),
+        _.bind(dataUtil.findVersionItem, null, items, state.from.replace('sys-perf', 'sys_perf')),
```

Comment:
Getting the following error:

angular.js:14700 TypeError: Cannot read property 'replace' of undefined
    at PerformanceDiscoveryCtrl.js?hash=25c124042c90257e517a2105182d2ad21d449338:59
    at angular.js:17051
    at m.$digest (angular.js:18233)
    at m.$apply (angular.js:18531)
    at l (angular.js:12547)
    at s (angular.js:12785)
    at XMLHttpRequest.y.onload (angular.js:12702) "Possibly unhandled rejection: {}"

Reply:
I can remove this call, it was to fix another issue, unrelated to this PR. I'll raise a separate ticket for it.